lib/active_model/serializer/associations.rb

```diff
@@ -76,10 +78,6 @@ def has_one(name, options = {})
         def associate(reflection)
           self._reflections = _reflections.dup
```

Comment:
Yeah, It doesn't seem to make sense, but it's pre-existing, and there's enough changes in here already